From: Baruch Siach <baruch@tkos.co.il>
Subject: [PATCH] poco: add the staging path to search path

Add the mysql headers and client libraries to the search path of the
preprocessor and the linker. The $MYSQL_LIBDIR / $MYSQL_INCIDR variables
must be set from the make command line.

[Peter: Remove host dirs, add MYSQL_INCDIR]
Signed-off-by: Baruch Siach <baruch@tkos.co.il>
--- 
--- poco-1.4.4-all-dist/Data/MySQL/Makefile	2012-09-20 01:30:42.000000000 +0300
+++ poco-1.4.4-all/Data/MySQL/Makefile	2012-10-11 14:36:10.343153420 +0200
@@ -8,8 +8,8 @@
 
 include $(POCO_BASE)/build/rules/global
 
-SYSLIBS += -L/usr/local/lib$(LIB64SUFFIX)/mysql -L/usr/lib$(LIB64SUFFIX)/mysql -L/usr/mysql/lib$(LIB64SUFFIX)/mysql -L/usr/local/mysql/lib$(LIB64SUFFIX) -lmysqlclient
-INCLUDE += -I/usr/local/include/mysql/ -I/usr/include/mysql -I/usr/mysql/include/mysql -I/usr/local/mysql/include
+SYSLIBS += -L$(MYSQL_LIBDIR) -lmysqlclient
+INCLUDE += -I$(MYSQL_INCDIR)
 SYSFLAGS += -DTHREADSAFE -DNO_TCL
 
 objects = Binder Extractor SessionImpl Connector \
